sotopia/ui/README.md

```diff
@@ -0,0 +1,162 @@
+# Sotopia UI
+
+## FastAPI Server
+
+The API server is a FastAPI application that is used to connect the Sotopia UI to the Sotopia backend.
+This could also help with other projects that need to connect to the Sotopia backend through HTTP requests.
+
+Here are some initial design of the API server:
+
+### Getting Data from the API Server
+
+#### GET /get/scenarios/{scenario_id}
+
+Get scenarios by scenario_id.
+parameters:
+- scenario_id: str
+
+returns:
+- scenarios: EnvironmentProfile
+
+
+#### GET /get/scenarios
+
+Get all scenarios.
+
+returns:
+- scenarios: list[EnvironmentProfile]
+
+#### GET /get/scenarios/{sceanrio_tag}
```

Comment:
how would the server tell the difference from `GET /get/scenarios/{scenario_id}`? Maybe change it to `GET /scenarios/get_by_tag/{scenario_tag}`

Reply:
or add a parameter get_by=id/tag?

Reply:
changed